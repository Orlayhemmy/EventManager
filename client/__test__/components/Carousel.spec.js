import React from 'react';
import renderer from 'react-test-renderer';
import expect from 'expect';
import Carousel from '../../components/Carousel/Container/Carousel';

describe('Carousel Component', () => {
  const props = {
    centerState: {
      centers: ['balmoral', 'andela']
    }
  };
  const wrapper = shallow(<Carousel {...props} />);
  it('should render without throwing error', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('check the length of the containing div in the component', () => {
    expect(wrapper.find('#carousel').length).toEqual(1);
  });
});
