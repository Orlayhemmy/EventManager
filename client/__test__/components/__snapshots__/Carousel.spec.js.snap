// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel Component should render without throwing error 1`] = `
<div
  className="col-lg-6 form"
  id="carousel"
>
  <h1>
    Our Centers
  </h1>
  <div
    className="carousel slide"
    data-ride="carousel"
    id="carouselExampleIndicators"
  >
    <ol
      className="carousel-indicators"
    >
      <li
        className="active"
        data-slide-to={0}
        data-target="#carouselExampleIndicators"
        key="0"
      />
      <li
        className=""
        data-slide-to={1}
        data-target="#carouselExampleIndicators"
        key="1"
      />
    </ol>
    <div
      className="carousel-inner"
    >
      <div
        className="carousel-item active"
        key="0"
      >
        <img
          className="d-block w-100 h-350"
        />
        <div
          className="carousel-caption d-none d-md-block"
        >
          <h5 />
          <p />
        </div>
      </div>
      <div
        className="carousel-item"
        key="1"
      >
        <img
          className="d-block w-100 h-350"
        />
        <div
          className="carousel-caption d-none d-md-block"
        >
          <h5 />
          <p />
        </div>
      </div>
    </div>
    <a
      className="carousel-control-prev"
      data-slide="prev"
      href="#carouselExampleIndicators"
      role="button"
    >
      <span
        aria-hidden="true"
        className="carousel-control-prev-icon"
      />
      <span
        className="sr-only"
      >
        Previous
      </span>
    </a>
    <a
      className="carousel-control-next"
      data-slide="next"
      href="#carouselExampleIndicators"
      role="button"
    >
      <span
        aria-hidden="true"
        className="carousel-control-next-icon"
      />
      <span
        className="sr-only"
      >
        Next
      </span>
    </a>
  </div>
</div>
`;
